src/index.html

```diff
@@ -11,12 +11,128 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
```

Comment:
The attributes of the link element should be formatted correctly. Each attribute should start on a new line with a 2-space indentation relative to the tag.